test/services/worker_task_tracking_repository.js

```diff
@@ -13,31 +13,44 @@ import {cleanDatabase, connectToMongo} from '../../src/utils/db_utils';
 import config from '../../config/config';
 
 import WorkerTaskTrackingRepository from '../../src/services/worker_task_tracking_repository';
+import sinon from 'sinon';
 
 const {expect} = chai;
 chai.use(chaiAsPromised);
 
-describe('Worker Log Repository', () => {
+describe('Worker Task Tracking Repository', () => {
   let db;
   let client;
   let storage;
+  let clock;
   const exampleTaskType = 'deliveringAmbrosia';
   const anotherExampleTaskType = 'castingLightnings';
   const fakeTaskId = 'hephaestusDidNotKnowAboutPompei';
+  const now = 15000000000;
 
   before(async () => {
     ({db, client} = await connectToMongo(config));
     storage = new WorkerTaskTrackingRepository(db);
+    clock = sinon.useFakeTimers(now);
   });
 
   afterEach(async () => {
     await cleanDatabase(db);
+    clock.restore();
```

Comment:
the calls to `useFakeTimers` and `clock.restore` are not symmetric (before vs. afterEach) -> only the first test case will be using the fake timer :(